saltstack/base/salt/prerequisites/pip.sls

```diff
@@ -24,15 +24,21 @@ update_python_pip2:
 
 update_python_pip3:
   cmd.run:
-    - name: pip3 install --upgrade --index=https://pypi.python.org/simple/ pip==8.1.2
-    - onlyif: pip3 -V
+    - name: python3 -m pip install --upgrade --index=https://pypi.python.org/simple/ pip==8.1.2
+    - onlyif: python3 -m pip -V
 
 {% endif %}
 
+{% if pillar['OS'] == 'redhat8' %}  
+install_pyyaml:
+  cmd.run:
+    - name: python3 -m pip install "PyYAML>=5.1" --ignore-installed 
```

Comment:
Suggestion:

```
install_pyyaml:
  pip.installed:
    - name: PyYAML
```
    
Also, we're installing PyYAML 6.0 by default either way - you can get rid of the whole "if" part too